src/v/cluster/service.cc

```diff
@@ -744,6 +747,24 @@ ss::future<partition_state_reply> service::get_partition_state(
     });
 }
 
+ss::future<controller_last_applied_reply> service::get_controller_last_applied(
+  controller_last_applied_request&&, rpc::streaming_context&) {
+    return ss::with_scheduling_group(get_scheduling_group(), [this]() {
+        return ss::smp::submit_to(0, [this]() {
```

Comment:
instead of `0` can we use the constant for controller leader?

Reply:
theres one for controller_stm not controller

Reply:
Should we add a constant for the controller or just use the `stm` one? I think it helps readability.

Reply:
Sounds good